src/api/test/models/article.test.coffee

```diff
@@ -87,7 +87,7 @@ describe "Article", ->
 
     it 'creates a lower score for an older article', ->
       @article.set 'published', true
-      @article.set 'published_at', new Date(2016, 4, 10)
+      @article.set 'published_at', new Date(2022, 4, 10)
```

Comment:
The one spec failure was legit, based on this hard-coded date. 

I just bumped the date instead of doing anything more clever, and the test passes now.